services/core/shared/core/compat/sdk_v5/accounts.js

```diff
@@ -133,7 +133,10 @@ const getAccountsFromCore = async (params) => {
 
 		await BluebirdPromise.map(
 			accounts.data,
-			async account => accountsCache.set(account.address, JSON.stringify(account)),
+			async account => {
+				const ttlMillisecs = 5 * 60 * 1000; // Expire cache entry in 5 mins
```

Comment:
do you think it would be better to get this value from config? in case if we need to increase or decrease we can ask devops to change and restart

Reply:
Sure, I'll add it as a config with a supporting environment variable.

Reply:
Done!